src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -72,6 +72,7 @@ void addONNXToZHighPasses(
   pm.addPass(onnx_mlir::createShapeInferencePass());
   pm.addPass(mlir::createCanonicalizerPass());
   pm.addNestedPass<func::FuncOp>(onnx_mlir::createConstPropONNXToONNXPass());
+  pm.addPass(onnx_mlir::createShapeInferencePass());
```

Comment:
Is this additional pass needed by this change or in general?

Reply:
It is needed in general like what you did to repeat "constprop->shape inference->...".